sql/test_data.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO tests (name, run_frequency_mins, method, url, headers, payload, status)
```

Comment:
Executing this currently leads to this error: 
```
Error invoking remote method 'DB_EXECUTE_CANCELLABLE_QUERY': error: null value in column "created_at" of relation "tests" violates not-null constraint
```
@mabbason how would you feel about adding a default value of now() to the created_at column? This might look like: 
```
...
created_at timestamp NOT NULL DEFAULT NOW(),
updated_at timestamp
...
```

Reply:
yeah, that's a better way